src/common/errors.js

```diff
@@ -7,15 +7,23 @@ function extendableBuiltin(cls) {
     cls.apply(this, arguments);
   }
   ExtendableBuiltin.prototype = Object.create(cls.prototype);
-  Object.setPrototypeOf(ExtendableBuiltin, cls);
-
+  // Object.setPrototypeOf not supported on Internet Explorer 9
+  /* eslint-disable */
+  Object.setPrototypeOf ? Object.setPrototypeOf(ExtendableBuiltin, cls) :
+    ExtendableBuiltin.__proto__ = cls;
+  /* eslint-enable */
   return ExtendableBuiltin;
 }
 
 class RippleError extends extendableBuiltin(Error) {
   constructor(message, data) {
     super(message);
-    this.name = this.constructor.name;
+
+    // hack for internet explorer
```

Comment:
Does lodash have any methods that might help for these? If not, it would be nice to extract all the "browser hacks" to a single file at src/common/browser-hacks.js to keep them out of our real code.